roles/manifest/README.md

```diff
@@ -0,0 +1,69 @@
+theforeman.foreman.manifest
+===============================================
+
+Upload Subscription Manifest to Katello
+
+Requirements
+------------
+
+- `theforeman.forman` collection
+
+Role Variables
+--------------
+
+- `server_url`: URL of the Foreman server.
+- `username`: Username accessing the Foreman server.
+- `password`: Password of the user accessing the Foreman server.
+- `validate_certs`: Whether or not to verify the TLS certificates of the Foreman server.
+- `organization`: Organization to upload Manifest to.
+- `manifest_path`: Where to find the Manifest on Foreman server. When using download_manifest, it is first downloaded to this location.
+- `download_manifest`: Whether to first download the Manifest from the Red Hat Customer Portal.
+- `manifest_uuid`: UUID of the Manifest to download. Most correspond to a Subscription Allocation defined at https://access.redhat.com/management/subscription_allocations
+- `rhsm_username`: Your username for the Red Hat Customer Portal. Required when download_manifest is True.
+- `rhsm_password`: Your password for the Red Hat Customer Portal. Required when download_manifest is True.
```

Comment:
These two are probably also valid as common variables that can go in the top level README

Reply:
What other role would use these, @ehelms ? (cloud_connector?)